package.json

```diff
@@ -81,6 +81,7 @@
     "eslint-plugin-loosely-restrict-imports": "^0.1.15",
     "eslint-plugin-react": "7.11.1",
     "file-loader": "2.0.0",
+    "focus-trap-react": "^6.0.0",
```

Comment:
@droshev, @jeffredodd :

focus-trap-react is a light-weight wrapper around `focus-trap` https://github.com/davidtheclark/focus-trap

Angular implementation: https://github.com/hassam7/ngx-focus-trap
Vue implementation: https://github.com/wovue/focus-trap

Reply:
Yeah I would agree that consistency across the difference platforms will help us keep the A11y focus trapping the same in all 3 libraries.

Reply:
should this be a `Dep`, rather than a `devDep`?

Reply:
Moved to dependencies